pulp_service/pulp_service/app/models.py

```diff
@@ -1,12 +1,15 @@
+from django.conf import settings
+
 from django.db import models
-from pulpcore.plugin.models import Domain
+from pulpcore.plugin.models import Domain, Group
+
+
 class DomainOrg(models.Model):
     """
     One-to-many relationship between org ids and Domains.
     """
-
-    org_id = models.CharField(db_index=True)
-    domain = models.ForeignKey(Domain, related_name="domains", on_delete=models.CASCADE, db_index=True)
-
-    class Meta:
-        unique_together = ("org_id", "domain")
+    org_id = models.CharField(null=True, db_index=True)
+    domain = models.ForeignKey(Domain, related_name="domains", on_delete=models.CASCADE)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL, related_name="users", on_delete=models.CASCADE
+    )
```

Comment:
I think these are indexed by default, but are we sure? Maybe just put `db_index=True` to be explicit (and sure)?

Reply:
> A database index is automatically created on the ForeignKey. You can disable this by setting [db_index](https://docs.djangoproject.com/en/3.2/ref/models/fields/#django.db.models.Field.db_index) to False. You may want to avoid the overhead of an index if you are creating a foreign key for consistency rather than joins, or if you will be creating an alternative index like a partial or multiple column index.

See https://docs.djangoproject.com/en/3.2/ref/models/fields/#foreignkey.